tools/generator/test/CreateXCSchemesTests.swift

```diff
@@ -0,0 +1,184 @@
+import PathKit
+import XcodeProj
+import XCTest
+
+@testable import generator
+
+class CreateXCSchemesTests: XCTestCase {
+    let filePathResolver = FilePathResolver(
+        internalDirectoryName: "rules_xcodeproj",
+        workspaceOutputPath: "examples/foo/Foo.xcodeproj"
+    )
+    let pbxTargetsDict: [TargetID: PBXTarget] =
+        Fixtures.pbxTargetsWithDependencies(
+            in: Fixtures.pbxProj(),
+            targets: Fixtures.targets
+        )
+
+    func assertScheme(
+        schemesDict: [String: XCScheme],
+        targetName: String,
```

Comment:
```suggestion
        targetID: TargetID,
```

Reply:
OK. However, it is not entirely necessary as the extension does not care what the key type is.

Reply:
Done.